gateway/pom.xml

```diff
@@ -66,6 +66,15 @@
       <artifactId>lombok</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-amqp</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.amqp</groupId>
+      <artifactId>spring-rabbit</artifactId>
+      <version>2.3.11</version>
+    </dependency>
```

Comment:
this is an older spring-rabbit version than the one (2.4.8) that comes as transitive with spring-boot 2.6.14 (transitive from spring-cloud:2021.0.7 from gateway's root pom). See https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-amqp/2.6.14

Is there a reason for the downgrade?